curriculum/challenges/english/25-front-end-development/lecture-working-with-file-systems/672ac37104dc2530a769e6a4.md

```diff
@@ -10,6 +10,58 @@ dashedName: what-are-best-practices-for-naming-files-for-web-applications
 
 Watch the lecture video and answer the questions below.
 
+# --transcript--
+
+What are best practices for naming files for web applications?
+
+As you start to build out your own web applications, you will need to be mindful of what you name your files.
+
+You will want to name your files in a way that is easy to understand and maintain.
+
+Let's take a look at some examples of good and bad file names for HTML files. Note that the `.html` extension in these examples tells us that this is an HTML file.
+
+Here is an example of bad file name: 
+
+```md
+index1234.html
+```
+
+By the name alone, it is not clear what the file is about. This file could be for a products page, blog page, or any other type of page but we don't know that from the file name.
+
+Here is an example of a better file name for an HTML file:
+
+```md
+about-us.html
+```
+
+By using a more descriptive name like `about-us.html`, it is clear what the file is about.
+
+You will also notice in this file name that there are no spaces. Instead, you use a hyphen, `-`, to separate the words. 
```

Comment:
```suggestion
You will also notice in this file name that there are no spaces. Instead, you use a hyphen, (`-`), to separate the words. 
```

Reply:
Previously added scripts use both parentheses and commas (not at the same time), so I think it's fine to leave it as is.